book/graphs/implementation.md

```diff
@@ -31,7 +31,10 @@ class Vertex(object):
         self.neighbors[neighbor] = weight
 
     def __str__(self):
-        return '{} neighbors: {}'.format(self.key, [x.key for x in self.neighbors])
+        return '{} neighbors: {}'.format(
```

Comment:
This must have been spilling over... the character limits I'm only enforcing in .py files as that's where my linter runs :smile: